scale/messaging/backends/sqs.py

```diff
@@ -0,0 +1,41 @@
+"""Backend supporting Amazon SQS"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import json
+import logging
+
+from messaging.backends.backend import MessagingBackend
+from util.aws import AWSCredentials, SQSClient
+
+logger = logging.getLogger(__name__)
+
+
+class SQSMessagingBackend(MessagingBackend):
+    """Backend supporting message passing via Amazon SQS"""
+
+    def __init__(self):
+        super(SQSMessagingBackend, self).__init__('sqs')
+
+        self._region_name = self._broker.get_broker()
```

Comment:
This is a bit hokey looking, but correct. Maybe I'll change the name of the getter to be clearer.